bundles/org.openhab.binding.airq/README.md

```diff
@@ -0,0 +1,222 @@
+# air-Q Binding
+
+The air-Q Binding integrates the air analyzer <a href="http://www.air-q.com">air-Q</a> device into the openHAB system.
```

Comment:
Markdown syntax is preferred over HTML in this document, so that you can read it also unrendered. Same for the images.
```suggestion
The air-Q Binding integrates the air analyzer [air-Q](http://www.air-q.com) device into the openHAB system.
```